lib/wrappers/transform-node.ts

```diff
@@ -23,11 +23,9 @@ export default class TransformNodeWrapper extends NodeWrapper {
     });
 
     if (this.nodeInfo.inoutFacade) {
-      this.callbackObject = {
-        ...this.nodeInfo.getCallbackObject(),
-        input: new FSMerger(this.nodeInfo.inputNodes),
-        output: new OutputWrapper(this.outputPath)
-      };
+      this.callbackObject = this.nodeInfo.getCallbackObject();
```

Comment:
can we unit test the shape of this callbackObject? I want to ensure this API is locked down.

Reply:
I have added a test to cover this `if` branch. I forgot to push the changes. :)